terraform/aws/modules/gitlab/gitlab-route53.tf

```diff
@@ -8,14 +8,44 @@ data "kubernetes_service" "gitlab_nginx" {
   ]
 }
 
+
+data "aws_elb" "gitlab_nginx_elb" {
+  # hostname have the name of the LB: internal-a6c7cc5f1622422ea8cb712fe8s21b48-995323897.us-east-1.elb.amazonaws.com need to get the name in order to ge tthe zone_id that will be used to create the A record with alias
+  name = split("-", data.kubernetes_service.gitlab_nginx.load_balancer_ingress.0.hostname).1
+  depends_on = [
+    helm_release.gitlab,
+  ]
+}
+
```

Comment:
this one can be replaced by:
`data "aws_elb_hosted_zone_id" "main" {}`

Reply:
reference: https://www.terraform.io/docs/providers/aws/d/elb_hosted_zone_id.html

Reply:
the problem is this elb is not created by terraform, i tried and did not work, we need to fetch it

Reply:
I thought it didn't matter as the zone IDs are the same for the LBs in each region.
https://docs.aws.amazon.com/general/latest/gr/elb.html
Let me check it quickly

Reply:
it is already deploy as is and working :)